README.md

```diff
@@ -39,8 +39,6 @@ The figure below illustrates the software flow through the architecture componen
 
 # Getting Started
 
-Learn how to configure and run the service by following these [instructions](./doc/setup.md). 
```

Comment:
If we are removing the link to the `setup.md` do we want to just get rid of the repetitive files themselves? I.e. `setup.md`, `custom-build.md`, `general-usage.md`, `running-guide.md`. Or is that a discussion for later when we potentially move the docs?

Reply:
For now we can keep these files until the migration is done for Edgex docs.